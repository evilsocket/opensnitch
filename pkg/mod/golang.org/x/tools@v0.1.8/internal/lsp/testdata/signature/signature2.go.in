package signature

func _() {
	Foo(//@signature("//", "Foo(a string, b int) (c bool)", 0)
}
