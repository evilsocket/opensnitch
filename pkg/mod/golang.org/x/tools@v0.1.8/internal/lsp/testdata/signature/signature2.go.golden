-- Foo(a string, b int) (c bool)-signature --
Foo(a string, b int) (c bool)

